mysql/assets/configuration/spec.yaml

```
@@ -58,7 +58,7 @@ files:
             value:
               type: number
               example: 3306
-              default: None
+              default: 0
```

```suggestion
              default: null
```

Maybe here too? Or should default be the example since it's option is enabled

---

sure!